code-reviews.md

```diff
@@ -0,0 +1,141 @@
+  Can you clarify?")
+- Avoid selective ownership of code. ("mine", "not mine", "yours")
+- Avoid using terms that could be seen as referring to personal traits.
+- Be explicit, people don't always understand your intentions online.
+- Be humble. ("I'm not sure - let's look it up.")
+- Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+- Don't use sarcasm.
+- Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
+  If they are, use them with aplomb.
+- Talk offline if there is too much back and forth. Post a follow-up comment
+  summarizing the discussion.
+- Praise team members when they create exemplary work or suggestions.
+
+### For code submitters
+
+- Link to the code review from the originating Jira task/issue, if applicable.
+- Remember that the code isn't you, don't get defensive when a reviewer is critical
+  of the code.
+- Seek to understand the reviewer's perspective.
+- Try to respond to every comment.
+- Be grateful for the reviewer's suggestions. ("Good catch, fixing in a4994ec")
+- Explain why the code exists. ("We need to work around these existing patterns")
+- Extract out-of-scope changes and refactorings into future Jira tasks/issues.
+- Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+- Wait to merge the branch until it has passed Continuous Integration testing.
+  (TDDium, TravisCI, etc.)
+- Merge once you feel confident in the code and its impact on the project.
+
+### For code reviewers
+
+- Understand why the code is necessary (bug, user experience, refactoring)
+- Seek to understand the author's perspective.
+- Clearly communicate which ideas you feel strongly about and those you don't.
+- Identify ways to simplify the code while still solving the problem.
+- Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about such-and-such here?")
+- If discussions turn too philosophical or academic, move the discussion to a new
+  issue or offline. In the meantime, the author has the final say on the current
+  implementation.
+- Sign off on the pull request with a :thumbsup: or "Ready to merge" comment.
+
+
+## Submitting code
+
+Before seeking a review, you should be able to check off each of the following:
+
+[ ] Changes are limited to a single goal (no scope creep)
+[ ] Code can be automatically merged (no conflicts)
+[ ] Code follows the standards laid out in this playbook
+[ ] Passes all existing automated tests
+[ ] New functions include new tests
+[ ] New functions are documented (with a description, list of inputs, and
+    expected output)
+[ ] Placeholder code is flagged
+[ ] Visually tested in supported browsers and devices
+[ ] Project documentation has been updated (including the "Pending" section of
```

Comment:
What do you think about `Unreleased` instead of `Pending` per http://keepachangelog.com/